src/DynamoCoreWpf/ViewModels/Core/NoteViewModel.cs

```diff
@@ -337,7 +337,7 @@ private void PinnedNodeModel_PropertyChanged(object sender, System.ComponentMode
             if (e.PropertyName == nameof(NodeModel.State)
                 || e.PropertyName == nameof(NodeModel.Position))
             {
-                var node = sender as NodeViewModel;
+                var node = sender as NodeModel;
```

Comment:
why is this actually needed? dosent look like you are actually using `node` for anything?